Loops_Lab.js

```diff
@@ -0,0 +1,210 @@
+    console.log("Fizz");
+  } else if (i % 5 === 0){
+    console.log("Buzz");
+  } else console.log(i);
+}
+console.log("----------------------------------------- \n");
+
+
+// Bonus1:
+// Write a program that would log the lyrics of the song 99 Bottles of Beer. This is the first verse of the song:
+// 99 bottles of beer on the wall, 99 bottles of beer.
+// Take one down, pass it around, 98 bottles of beer on the wall.
+// This verse is repeated, each time with one less bottle, until the number of bottles is 0. When the number of bottles is 2, the verse is:
+//
+// 2 bottles of beer on the wall, 2 bottles of beer.
+// Take one down, pass it around, 1 bottle of beer on the wall.
+// In the last line, the word bottles (plural), is replaced with bottle (singular)
+//
+// When the number of bottles is 1, the verse is:
+//
+// 1 bottle of beer on the wall, 1 bottle of beer.
+// Take one down, pass it around, No more bottle of beer on the wall.
+//
+let lyricsNum = 5; // did it for just 5 for an easier display
+
+for (i = lyricsNum; i >= 1; i--){
+  switch (i){
+    case 1:
+      console.log("1 bottle of beer on the wall, 1 bottle of beer.\n",
+                  "Take one down, pass it around, No more bottle of beer on the wall.\n \n");
+    break;
+    case 2:
+      console.log("2 bottles of beer on the wall, 2 bottles of beer.\n",
+                  "Take one down, pass it around, 1 bottle of beer on the wall.\n \n");
+    break;
+    default: console.log(i, " bottles of beer on the wall, ", i, " bottles of beer.\n",
+                          "Take one down, pass it around, ", i-1, " bottles of beer on the wall.\n \n");
+  }
+}
+console.log("----------------------------------------- \n");
+
+
+// Bonus2 :
+// Use the assignGrade function (given below). Write a function that logs each
+// number from 60 - 100 along with its corresponding letter score. Exp For each
+// number from 81 to 90, log B, like so:
+//
+// 60 - F
+// ...
+// 81 - B
+// 82 - B
+// 83 - B
+// ...
+// 93 - A
+// 94 - A
+// ...
+function assignGrade(score) {
+    if (score > 90) {
+        return 'A';
+    } else if (score > 80) {
+        return 'B';
+    } else if (score > 70) {
+        return 'C';
+    } else if (score > 65) {
+        return 'D';
+    } else {
+        return 'F';
+    }
+}
+//  Hint
+
+for (let i = 60; i <= 100; i++){
+  console.log("The score ", i, " is grade :", assignGrade(i));
+}
+console.log("----------------------------------------- \n");
+
+// Bonus3:
+// Given an integer N draw a square of N x N asterisks.
+let N = 5;
+let ast = "";
+
+for (let i = 1; i <= N; i++){
+  ast += "* ";
+}
+  for (let j = 1; j <= N; j++){
+      console.log(ast);
+}
+console.log("----------------------------------------- \n");
+
+// Bonus4:
+// Multiplication table with only two loops
+for (let i = 1; i <= 10; i++){
+  console.log("\n", i, "Multiplication table");
+  for (let j = 1; j <= 10; j++){
+    console.log(i, " x ", j, " = ", i*j);
+  }
```

Comment:
Overall good demonstration of loops! Other than aforementioned feedback, everything looks good. Keep up the good work!

Reply:
Thank you for all the feedback